src/clients/inst_autosetup.rb

```diff
@@ -276,6 +278,12 @@ def main
 
       Progress.NextStage
 
+      # Importing security settings
+      autosetup_security
+      return :abort if Popup.ConfirmAbort(:painless) if UI.PollInput == :abort
```

Comment:
Double `if`. 

```suggestion
      return :abort if UI.PollInput == :abort && Popup.ConfirmAbort(:painless)
```

BTW I bet Rubocop would complain here loudly. :wink: